sprox/mg/provider.py

```diff
@@ -406,7 +406,10 @@ def update(self, entity, params, omit_fields=None):
     def delete(self, entity, params):
         """Delete an entry of typeentity which matches the params."""
         obj = self.get_obj(entity, params)
-        obj.delete()
+        try:
+            obj.delete()
+        except AttributeError:
```

Comment:
The `AttributeError` is probably a side effect of applying `delete` to None, I think that instead of catching the attribute error which might hide bugs it's probably best to check if `obj` is `None` or whatever `get_obj` is returning when the object doesn't exist anymore.